ecl/wutest/wujobqtest2.cpp

```diff
@@ -56,6 +56,7 @@ bool switchWorkunitQueue(const char *wuid, const char *cluster)
     Owned<IWorkUnit> wu = factory->updateWorkUnit(wuid);
     if (!wu)
         return false;
-    return wu->switchThorQueue(cluster, &switcher);
+    VStringBuffer item("*/%s/*", wuid);
+    return wu->switchThorQueue(cluster, &switcher, item);
```

Comment:
Should pass nullptr as item - otherwise it will only match 1 item (although without parallel workflow I doubt there is ever >1)

Reply:
yes, that's a mistake, will change.